src/components/organization/organization.resolver.ts

```diff
@@ -87,6 +103,30 @@ export class OrganizationResolver {
     return true;
   }
 
+  @Mutation(() => Organization, {
+    description: 'Add a location to a organization',
+  })
+  async addLocationToOrganization(
+    @Session() session: ISession,
+    @Args('organizationId', { type: () => ID }) organizationId: string,
+    @Args('locationId', { type: () => ID }) locationId: string
```

Comment:
Could you do these like:
https://github.com/SeedCompany/cord-api-v3/blob/229348e2c1d577051ba770a49560e7c37c8a7997/src/components/authorization/authorization.resolver.ts#L34
https://github.com/SeedCompany/cord-api-v3/blob/229348e2c1d577051ba770a49560e7c37c8a7997/src/components/authorization/authorization.resolver.ts#L13-L16
That way IDs are validated.